src/useAsyncEffect.js

```diff
@@ -5,7 +5,7 @@ const hasChanged = (lastValue, thisValue) => JSON.stringify(lastValue) !== JSON.
 
 const clone = o => JSON.parse(JSON.stringify(o))
 
-export default ({ name, options, args = [] }) => {
+export default ({ name, options }, ...args) => {
```

Comment:
Do we need to worry about supporting the other way for easy upgrades?  Or assume that no one is using the older versions?

Reply:
that's a fair question, breaking changes are a pain.  it's hard to say who this might adversely effect though

Reply:
To my knowledge, nothing is using this yet.